projects/framework-maya/extensions/plugins/maya_save_to_temp_finalizer.py

```diff
@@ -22,6 +22,7 @@ def run(self, store):
             raise PluginExecutionError(
                 "Can't query the scene type of the current opened scene"
             )
+        # TODO: Possible missing "." prefix on scene_type
```

Comment:
> But in any case, to align our default code, I think we should always provide the extensions without the "." ("mb" instead of ".mb") so whenever we use them in ui related stuff like in a selectable combo box, we show the clean extension ("mb") name and not the ".mb".

Did you also update the current code in PS to remove the dot? 
I have added a task for nuke to take later: https://dev.ftrackapp.com/#slideEntityId=824cd33d-aa78-4a53-8721-f55dc0e9cf87&slideEntityType=task&view=tasks&itemId=projects&entityId=dc34b754-79e8-11e3-b4d0-040102b7e101&entityType=show

Reply:
Yes, the widget is now rid of dots in formats

Reply:
Please remove this change from the file.